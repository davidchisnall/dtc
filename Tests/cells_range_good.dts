/dts-v1/;

/ {
	foo8 = /bits/ 8 <0 0xff (-1) (-0x80)>;
	foo16 = /bits/ 16 <0 0xffff (-1) (-0x8000)>;
	foo32 = /bits/ 32 <0 0xffffffff (-1) (-0x80000000)>;
	foo64 = /bits/ 64 <0 0xffffffffffffffff (-1) (-0x8000000000000000)>;
}
