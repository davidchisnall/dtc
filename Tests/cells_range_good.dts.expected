/dts-v1/;

/  {

	foo8 = <0xffff80>;
	foo16 = <0xffff 0xffff8000>;
	foo32 = <0x0 0xffffffff 0xffffffff 0x80000000>;
	foo64 = <0x0 0x0 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0x80000000 0x0>;
};
